src/pytom_tm/extract.py

```diff
@@ -250,6 +250,22 @@ def extract_particles(
         # mask should be larger than zero in regions of interest!
         score_volume[tomogram_mask <= 0] = 0
 
+    if particle_diameter is not None:
+        particle_radius_px = int((particle_diameter / 2) / job.voxel_size)
+    elif job.particle_diameter is not None:
+        particle_radius_px = int((job.particle_diameter / 2) / job.voxel_size)
+        logging.info(
+            "No particle diameter was provided, so using the diameter "
+            "specified previously to mask out areas around peaks. Take care for "
+            "strongly elongated particles as it might prevent correct "
+            "annotation when they arrange parallel to each other and close together."
+        )
```

Comment:
This elif is not hit by the unit-tests, please alter one of the tests to grab the particle diameter from the previously run job.